crates/optimism/node/src/node.rs

```diff
@@ -270,6 +279,67 @@ where
     }
 }
 
+/// A regular optimism evm and executor builder.
+#[derive(Debug, Clone)]
+#[non_exhaustive]
+pub struct OpAddOnsBuilder<N> {
+    /// Sequencer client, configured to forward submitted transactions to sequencer of given OP
+    /// network.
+    sequencer_client: Option<SequencerClient>,
+    /// List of addresses that _ONLY_ return storage proofs _WITHOUT_ an account proof when called
+    /// with `eth_getProof`.
+    storage_proof_only: Vec<Address>,
+    _marker: PhantomData<N>,
```

Comment:
hmm, we can probably get rid of this entirely if we make this accept the type on `fn build` instead
then this doesn't need a generic at all